csexec-symbiotic.sh

```diff
@@ -47,4 +47,4 @@ fi
   /usr/bin/symbiotic2cs > "$LOGDIR/pid-$$.out.conv"
 
 # Continue
-exec $(/usr/bin/csexec --print-ld-exec-cmd) "${ARGV[@]}"
+exec $(/usr/bin/csexec --print-ld-exec-cmd "$CSEXEC_ARGV0") "${ARGV[@]}"
```

Comment:
Wouldn't it be more readable to use something like this?
```suggestion
exec $(/usr/bin/csexec --print-ld-exec-cmd) --argv0 "$CSEXEC_ARGV0" "${ARGV[@]}"
```

Reply:
This is not the same.  We need to append `--argv0 ...` only if the system dynamic linker actually supports the option (which csexec checks at build-time).  We also need to gracefully handle the case where `$CSEXEC_ARGV0` is for whatever reason unset ( see commit 96abba8fe534a5e9c3ce5226f9020fb2e2528984).

Reply:
Thanks for the explanation! What about `CSEXEC_ARGV0` with spaces?

Reply:
We do not support spaces in `argv[0]`.  The interface of `csexec --print-ld-exec-cmd` is too simple for that.  Btw. this reminded me why my fix for `argv[0]` was not effective on RHEL-7 servers.  I have opened pull request csutils/csmock#57 to fix it.

Reply:
In theory, `csexec --print-ld-exec-cmd` could be improved at the cost of making it more dependent on a shell interpreter:
```diff
--- a/src/csexec.c
+++ b/src/csexec.c
@@ -64,13 +64,13 @@ static int print_help(void)
 // print a command prefix that can be used to invoke dynamic linker explicitly
 static int print_ld_exec_cmd(const char *argv0)
 {
     printf("%s", LD_LINUX_SO " --preload " LIBCSEXEC_PRELOAD_SO);
 #if LD_LINUX_SO_TAKES_ARGV0
     if (argv0)
-        printf(" --argv0 %s", argv0);
+        printf(" --argv0 '%s'", argv0);
 #else
     (void) argv0;
 #endif
     printf("\n");
     return EXIT_SUCCESS;
 }
```

Reply:
Hmm, let's then leave it as it is. We can always return to this problem if we find a program that actually depends on multi-word `argv[0]`.

Reply:
Sure.  Thanks for review!